src/main.js

```diff
@@ -51,39 +58,46 @@ class WebdriverPercy {
     browser.percy.environment = new Environment(process.env);
     browser.percy.percyClient = new PercyClient({ token, apiUrl, clientInfo });
 
-    browser.addCommand('__percyReinit', function async() { // eslint-disable-line prefer-arrow-callback
+    browser.addCommand('__percyReinit', () => {
+      // eslint-disable-line prefer-arrow-callback
       browser.percy = { assetLoaders: [] };
       browser.percy.environment = new Environment(process.env);
       browser.percy.percyClient = new PercyClient({ token, apiUrl, clientInfo });
     });
 
-    browser.addCommand('percyFinalizeBuild', function async() { // eslint-disable-line prefer-arrow-callback
```

Comment:
Does the asynchronous have special meaning here and above?

Reply:
Yes my understanding is that if we name it `async` we need to return a promise, otherwise it's expected to be a synchrnous call - see http://webdriver.io/guide/usage/customcommands.html

Since we do api requests async sounds to make sense